provider/k8s/system.go

```diff
@@ -35,7 +36,7 @@ func (p *Provider) SystemGet() (*structs.System, error) {
 
 	s := &structs.System{
 		Domain:   fmt.Sprintf("router.%s", p.Domain),
-		Name:     p.Name,
+		Name:     p.RackName,
```

Comment:
From the changes i can understand. This var is set only when you ran the `mv` command.
will this have proper value for freshly installed rack of this version?

Reply:
it will also be set on rack install, the problem with the [rack variables](https://github.com/convox/convox/blob/bd16c3cac73cb4bd264db88809fa9eeb8115e558/terraform/system/aws/variables.tf#L80) is that we can't change or all resources will be destroyed, so the RackName will be the actual rack name and name is the "original" name.

Reply:
little bit confused about the rack name and original name?